content/source/docs/enterprise/logging/index.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "enterprise2"
+page_title: "Terraform Enterprise Logging"
+sidebar_current: "docs-enterprise2-home"
+---
+
+## Terraform Enterprise Logs
+
+This document contains information about interacting with Private Terraform Enterprise logs.
+
+### Audit Logs
+
+As of Private Terraform Enterprise release v201802-1, audit logging is available in Private Terraform Enterprise. 
+
+#### What will be logged?
+
+The audit log will be updated when any resource managed by Terraform Enterprise is changed. Read requests will be logged for resources deemed sensitive. These include:
+
+  * Authentication Tokens
+  * Configuration Versions
+  * Policy Versions
+  * OAuth Tokens
+  * SSH Keys
+  * State Versions
+  * Users
+  * Vars
+
+
+When requests occur, these are the pieces of information that will be logged:
+
+  1. The actor
+		* Users (including IP address and token used)
+		* Version Control System users (identified in webhooks)
+		* Service accounts
+		* Terraform Enterprise
+	2. The action
+		* Reading sensitive resources
+		* Creation of new resources
+		* Updating existing resources
+		* Deletion of existing resources
+		* Additional actions as defined in /actions/* namespaces
+		* Webhook API calls
+	3. The target of the action (any resource exposed by APIv2)
+	4. The time that the action occurred
+	5. Where the action was taken (web/API request, background job, etc.)
+
+#### How will log entries be written?
+
+Log entries are in JSON, just like other terraform logs. Log formatting will look like this: 
+
+```
+{"time":"2017-12-19T15:54:33.792Z","action":"plan","resource":"run","resource_id":"run-RgPD6h3tdpj9GaHG","actor":"gh-webhooks-tfe-dev-v2"}
+{“time":"2017-12-19T16:56:18.623Z","resource":"run",action":"apply","resource_id":"run-RgPD6h3tdpj9GaHG","actor":"johnsmith"}
```

Comment:
There's two stray smartquotes in here, making it invalid json. Good ol' “smart” quotes. -_-